pir/cpp/server.h

```diff
@@ -120,6 +121,22 @@ class PIRServer {
       const seal::Ciphertext& ct, const size_t num_items,
       const seal::GaloisKeys& gal_keys) const;
 
+  /**
+   * Extension of oblivious_expansion to multiple ciphertexts. This allows
+   * selection vectors that are larger then poly_modulus_degree to be used. The
+   * output of the expansion of each ciphertext is concatenated to form the
+   * results of this function.
+   *
+   * @param[in] cts List of ciphertexts to use as input to the expansion
+   * @param[in] num_items Total number of items to expand
```

Comment:
This is confuding: num_items is actually the (expected) size of the output vector.

Reply:
What would you suggest as a clearer name? This seems straightforward to me, but clearly not to everyone

Reply:
Maybe it should be "to expand to"? The current wording seems to say that this is the size of `cts`.

Reply:
I've changed it to total_items and added some text explaining it a bit more. Best short name I could think of :)